Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:latest
+FROM python:3.9
```

Comment:
In python3.10 the [collections abstract base classes](https://docs.python.org/3.9/library/collections.html#module-collections) was moved to collections.abc and this breaks the pyindi install. This will need to be fixed in pyindi.

Reply:
i'm starting the shift to 3.10 now that the ecosystem is catching up, but fine to pin to 3.9.